src/theme/components/blog-layout/blog-layout.scss

```diff
@@ -6,15 +6,21 @@ div[class^="authorCol_"] {
   background: red;
 }
 
+.markdown {
+  margin-bottom: 30px;
+
+   p {
+     max-width: 700px;
+   }
+}
+
 #__docusaurus {
-  .blog-wrapper {
+  .main-wrapper {
     .container {
       padding: 0 !important;
       margin: 0;
       max-width: initial;
-      .markdown {
-        margin-bottom: 30px;
-      }
+
```

Comment:
This seems to mess with stuff on the careers pages?

Reply:
have separated into new PR for this code.

Reply:
@bigandy looks like it's still here in this one?

Reply:
noooo. Sorry. Too many branches going on at the moment!

Reply:
done. and also used :where() in the css